src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -148,10 +146,6 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
     @FeatureToggle(Feature.PROGRAMMING_EXERCISES)
     public ResponseEntity<ProgrammingExercise> setupProgrammingExercise(@RequestBody ProgrammingExercise programmingExercise) {
         log.debug("REST request to setup ProgrammingExercise : {}", programmingExercise);
-        if (programmingExercise == null) {
```

Comment:
Can never be null, because Spring does not allow `null` values for `@RequestBody` (default behavior).